tests/hotplug/tolerations.go

```diff
@@ -0,0 +1,124 @@
+package hotplug
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+	"kubevirt.io/kubevirt/pkg/libvmi"
+	"kubevirt.io/kubevirt/pkg/pointer"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	. "kubevirt.io/kubevirt/tests/framework/matcher"
+	"kubevirt.io/kubevirt/tests/libkubevirt"
+	"kubevirt.io/kubevirt/tests/libvmifact"
+	"kubevirt.io/kubevirt/tests/libwait"
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+var _ = Describe("[sig-compute][Serial]VM Tolerations", decorators.SigCompute, decorators.VMLiveUpdateFeaturesGate, Serial, func() {
+	var (
+		virtClient kubecli.KubevirtClient
+	)
+	BeforeEach(func() {
+		virtClient = kubevirt.Client()
+		kv := libkubevirt.GetCurrentKv(virtClient)
+		kv.Spec.Configuration.VMRolloutStrategy = pointer.P(v1.VMRolloutStrategyLiveUpdate)
+		testsuite.UpdateKubeVirtConfigValue(kv.Spec.Configuration)
+	})
+
+	Context("Updating VMs tolerations", func() {
+		It("should successfully live update tolerations", func() {
+
+			By("Creating a running VM")
+			vmi := libvmifact.NewGuestless(libvmi.WithNamespace(testsuite.GetTestNamespace(nil)))
```

Comment:
`libvmi.WithNamespace(testsuite.GetTestNamespace(nil)` is smell.

Reply:
I think that the suggestion is to move the `testsuite.GetTestNamespace(nil)` in the creation param:
```
vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(nil)).Create(context.Background(), vm, metav1.CreateOptions{})
```
in this case you should be able to drop the `libvmi.WithNamespace` option

Reply:
Thank you Federico :)